### ISSUE TITLE
Interface 'WebviewTag' incorrectly extends interface 'HTMLElement'.

### ISSUE THREAD

**Issue by k8w** [electron/electron#11447]
* Electron version: 1.7.9
* Operating system: Win10

### Expected behavior
TypeScript Compile Success

### Actual behavior
TypeScript version is 2.6.2, get compile error below:
> node_modules/electron/electron.d.ts(5390,13): error TS2430: Interface 'WebviewTag' incorrectly extends interface 'HTMLElement'.
  Types of property 'addEventListener' are incompatible.
    Type '{ (event: "load-commit", listener: (event: LoadCommitEvent) => void, useCapture?: boolean | undef...' is not assignable to type '{ <K extends "error" | "waiting" | "progress" | "ended" | "change" | "input" | "select" | "abort"...'.
      Types of parameters 'listener' and 'listener' are incompatible.
        Type 'EventListenerOrEventListenerObject' is not assignable to type '(event: LoadCommitEvent) => void'.
          Type 'EventListenerObject' is not assignable to type '(event: LoadCommitEvent) => void'.
            Type 'EventListenerObject' provides no match for the signature '(event: LoadCommitEvent): void'.

### How to reproduce

1. make a new typescript project
1. tsconfig.json is below:
```json
{
  "compilerOptions": {
    "target": "es5",
    "module": "commonjs",
    "lib": ["es6", "dom", "scripthost"],
    "outDir": "./dist",
    "strict": true,
    "noUnusedLocals": true,
    "typeRoots": ["./typings", "./node_modules/@types"]
  }
}
```
1. `tsc` and get error

**Comment by mykter** [comment]
Also occurs with electron 1.8.1. Started happening with the upgrade to typescript 2.6.2.

**Comment by mykter** [comment]
Just as note as I see there's a platform/windows label attached to this: it happens on Linux too.

**Comment by mykter** [comment]
Oh looks like this is a dupe of #10870 which in turn is fixed in https://github.com/electron/electron-typescript-definitions/issues/79 which will get released in electron 1.8.2

**Comment by codebytere** [comment]
@mykter i'm going to close this as a fix has been provisioned to be released in a near-future version :)
